react/features/participants-pane/components/breakout-rooms/components/native/CollapsibleRoom.js

```diff
@@ -46,23 +47,19 @@ export const CollapsibleRoom = ({ room, searchString }: Props) => {
         = `${room.name
     || t('breakoutRooms.mainRoom')} (${roomParticipantsNr})`;
 
-    // Regarding the fact that we have 3 sections, we apply
-    // a certain height percentage for every section in order for all to fit
-    // inside the participants pane container
-    const containerStyle
-        = roomParticipantsNr > 2 && styles.collapsibleRoomContainer;
-
     return (
         <CollapsibleList
-            containerStyle = { containerStyle }
             onLongPress = { _openContextMenu }
             title = { title }>
             <FlatList
                 bounces = { false }
                 data = { Object.values(room.participants || {}) }
                 horizontal = { false }
                 keyExtractor = { _keyExtractor }
-                // eslint-disable-next-line react/jsx-no-bind
+
+                // For FlatList as a nested list of any other FlatList or SectionList
+                // we have to pass a unique value to this prop
+                listKey = { _keyExtractor }
```

Comment:
This needs to be a string like "fooList" or something: https://reactnative.dev/docs/virtualizedlist#listkey not a function. Or am I missing something?

Reply:
But the function return a string, right? I followed this thread
https://stackoverflow.com/questions/49033549/invariant-violation-a-virtualizedlist-contains-a-cell-which-itself-contains-mor